rate_hertz: 1000

joints:
  - fl_joint1
  - fl_joint2
  - fl_joint3
  - fr_joint1
  - fr_joint2
  - fr_joint3
  - br_joint1
  - br_joint2
  - br_joint3
  - bl_joint1
  - bl_joint2
  - bl_joint3
